collector/database/classes/collector.php

```diff
@@ -85,7 +86,8 @@ public function get_metrics($metricnames = null, $since = false): array {
                   FROM {cltr_database_metrics}
                  WHERE name $clause
                  $starting
-               ORDER BY time asc";
+               ORDER BY time asc
```

Comment:
get_records_sql has params for order and limit

Reply:
Updated